packages/CollabCodeEdit-Core.package/CCEEdit.class/instance/editContent..st

```diff
@@ -1,3 +1,3 @@
 accessing
-editContent: anObject
-	editContent := anObject
\ No newline at end of file
+editContent: anItem
```

Comment:
`anItem` is no more descriptive that `anObject`

Reply:
Apparently, `item` is a special term used in conjunction w/ Squeak events. I would thus argue that it is.